docs/guides/production.md

```diff
@@ -0,0 +1,296 @@
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection 'upgrade';
+        proxy_set_header Host $host;
+        proxy_cache_bypass $http_upgrade;
+    }
+
+    location /assets/ {
+        alias /path/to/application/__clientTemp/build/;
+        gzip on;
+        gzip_types application/javascript text/css;
+
+        expires 30d;
+    }
+}
+```
+
+- `server_name` defines what hostname to respond to.
+- `proxy_pass` tells `nginx` to proxy all HTTP calls to the ExpressJS instance of `react-server`.  This setting assumes
+that `nginx` and `react-server` are running on the same machine.
+- `location /assets/ {}` tells `nginx` to server all assets directly from the `__clientTemp/build` directory that
+`react-server` creates when it compiles the application.  It also does some compression as well.
+
+Further enhancements can/should be made on the `nginx` side to account for other needs, but this is a basic working
+example.  At this point, `nginx` handles all requests on port 80 and proxies it to `react-server` running inside `ExpressJS`
+on port 3000.
+
+### `Apache`
+(instructions needed)
+
+# Static Assets
+This section deals with how to serve static assets alongside `react-server`.  Most likely, these should be served outside
+of ExpressJS using either an HTTP/HTTPS fronting server or through a CDN.  Both methods will be discussed.  Additionally,
+the two types of static assets that will be discussed in this section are: generated and referenced.
+
+A generated asset is one that `react-server` "compiles" for you.  Your application and any imported assets will be compiled
+by `react-server` and loaded automatically.  The default configuration is to use the "javascript server" on port 3001
+that is included with `react-server`.  Yes, that's right, there are two servers that `react-server` runs: ports 3000
+and 3001.  This is probably not what you want in production.
+
+A referenced asset would be something that your application links to, but `react-server` doesn't know about.  This might
+be a logo image that you link to with an `<img>` tag.
+
+## Served through a Fronting Server
+The example fronting server configurations show how to configure a fronting HTTP/HTTPS server to serve the static assets
+generated by `react-server` after compilation.  Simply point the fronting server to the appropriate `__clientTemp/build`
+directory and you're all set.
+
+If you have additional, referenced assets that you want to be served by your fronting server, you can add an addition
+`location {}` block to point the server to that directory as well.
+
+All of this relies on proper configuration of `react-server` as mentioned in the example `.reactserverrc` file at the
+beginning of this guide.  Because `react-server` loads your compiled assets for you, you need to tell it where to find the assets.
+The `jsUrl` setting at the beginning of this guide does just that.  If you use `jsUrl: /assets/`, then `react-server` will
```

Comment:
I fixed it by adding a link to the section where that example is.  Not perfect, but I figured it works for now.